edb/edgeql-parser/src/parser.rs

```diff
@@ -225,6 +254,7 @@ pub struct Terminal {
 pub struct Production<'a> {
     pub id: usize,
     pub args: &'a [CSTNode<'a>],
+    pub inlined_ids: Option<&'a [usize]>,
```

Comment:
Document what that is for? It's because we need to know inlined ids for error processing?